doc/chromebooks/luks-installation.md

```diff
@@ -0,0 +1,433 @@
+# - mt7: dtb-${kver}/mt8173-*.dtb
+# - mt8: dtb-${kver}/mt8183-*.dtb
+# - cbq: dtb-${kver}/sc7180-trogdor-*.dtb
+# - generic: dtb-${kver}/*.dtb
+#Note. you can specufy only dtb for your specyfic device to reduce space
+#even further but lose portability. PLS BE CAREFUL, if you select wrong
+#one the device will likely no work
+
+ls dtb-${kver}/*.dtb | xargs printf " -b %s" | xargs mkimage -D "-I dts -O dtb -p 2048" -f auto -A arm64 -O linux -T kernel -C lzma -a 0 -d Image.lzma -i initrd.img.xz kernel.itb
+vbutil_kernel --pack vmlinux.kpart --keyblock /usr/share/vboot/devkeys/kernel.keyblock --signprivate /usr/share/vboot/devkeys/kernel_data_key.vbprivk --version 1 --config cmdline --bootloader bootloader.bin --vmlinuz kernel.itb --arch arm
+cp -v vmlinux.kpart "/boot/vmlinux.kpart-initrd-${kver}"
+
+#cleanning up
+rm -f Image Image.lzma initrd.img.xz bootloader.bin kernel.itb vmlinux.kpart
+
+
+#final output
+
+#getting where root is mounted from, /dev/mmcblk0p4 or /dev/sda4 or /dev/mapper/encrypted
+root=$(findmnt -n -o SOURCE /)
+
+#encrypted partitions are inside /dev/mapper/
+if echo $root | grep -q "mapper"; then
+  rawname=$(echo $root | sed 's/\/dev\/mapper\///') #first we remove useless part
+  root=/dev/$(lsblk -no NAME --raw | grep -B1 $rawname | head -n 1) #then we get partition before encrypted on in lsblk
+fi
+
+#obtains the /dev/mmcblk1p or /dev/sda or similar
+kpart=$(echo $root | sed 's/.$//')
+
+#obtains parent disk, since sometimes there is p it otherwise would be tricky
+pdisk=$(lsblk -no PKNAME $root | head -n 1)
+
+#printing results
+echo ""
+echo "for single boot only testing:"
+
+#echo "IMPORTANT: please double check your mmcblk device name beforehand"
+#already take care of above
+
+echo ""
+echo "  dd if=/boot/vmlinux.kpart-initrd-${kver} of=${kpart}2"
+echo "  cgpt add -i 2 -S 0 -T 1 -P 15 /dev/${pdisk}"
+echo ""
+echo "to always boot this kernel after successful testing:"
+echo ""
+echo "  dd if=/boot/vmlinux.kpart-initrd-${kver} of=${kpart}1"
+echo ""
+
+```
+</details>
+
+you can also find it [here](./kernel-and-initrd.sh)
+
+make it executable
+```
+chmod +x /boot/kernel-and-initrd.sh
+```
+and run once
+```
+/boot/kernel-and-initrd.sh
+```
+
+after script is done you need to flash the "to always boot this kernel ..."
+
+_Note. if you want to modify kelner cmdline, after first run the script will generete a cmdline file at /boot/cmdline , you can edit it and rerun script again then flash the image, ofc you can also do that later without chroot if you boot into actual system_ 
+
+_Note. we're flashing this image to always boot because when at first non of partitions here contain kernel with proper initramfs, later when upgrading the kelner or cmdline, it's a good idea to test the kelner first so you don't brick you system_
+
+_Note. if something goes wrong and you can't see what becauseof splash screen, you can remove ```quiet splash``` from /boot/cmdline and rerun the script, then reflash_
```

Comment:
missing space
```suggestion
_Note. if something goes wrong and you can't see what because of splash screen, you can remove ```quiet splash``` from /boot/cmdline and rerun the script, then reflash_
```